test/test_lesbrary_forward_map.jl

```diff
@@ -68,7 +68,7 @@ using OceanTurbulenceParameterEstimation.Parameters: unconstrained_prior, transf
         Qᵇ = simulation.model.tracers.b.boundary_conditions.top.condition
         N² = simulation.model.tracers.b.boundary_conditions.bottom.condition
 
-        simulation.Δt = 10.0
+        simulation.5 = 10.0
```

Comment:
```suggestion
        simulation.Δt = 5.0
```

We should remove the mixing length changes and use default CATKE here.

Also we should regrid to a coarser grid.

Reply:
In this pr?

Reply:
Yes, because this PR updates Oceananigans and therefore uses a different version of CATKE than the version that was used when we wrote this test. That's why the stability has changed. The meaning of the parameters is also completely different.

Reply:
Luckily the latest version of Oceananigans has sensible defaults for CATKE (unlike previously, when the parameters were calibrated to _another code_ (OceanTurb), which is why we introduced a non-default version in these tests.